evergreen/eg_app/consumers.py

```diff
@@ -9,9 +9,11 @@
 from channels.generic.websocket import AsyncWebsocketConsumer
 
 # from channels.layers import get_channel_layer
-from django.apps import apps
+# from django.apps import apps
 from django.core.files.base import ContentFile
 
+from eg_app.models import Post
+
```

Comment:
I forget what it was but there was some reason this didn't work for I think it was @sumanyuj?

Did you test this at runtime? Maybe it was that import order thing you fixed in asgi

Reply:
OK yeah found the original discussion and that was the problem. I'm on mobile rn so can't confirm myself, but should be good

https://github.com/WeirdAlex03/Project-EverGreen/pull/50#discussion_r1862962724

Reply:
Yes I fixed that with the asgi.py import order. I looked up the problem and found a Reddit page that discussed it and pointed to that documentation. You have to initialize the django asgi application before importing any models that belong to that asgi app so that the app's registry can be populated first (my understanding).